subresource-loading.bs

```diff
@@ -211,12 +213,17 @@ bundle=] |web bundle| and [=fetch params=] |fetch params|:
    version of `1`, but this specification tracks what’s actually implemented in
    browsers, which still uses draft versions.
 
-1. Let |response| be the result of running <a spec="fetch">HTTP-network-or-cache
-   fetch</a> given |fetch params|.
+1. [=Fetch=] |request| with [=processResponse=] algorithm set to [=process web
+   bundle response=] which is partially applied with |web bundle|.
 
    Note: Chromium's current implementation doesn't allow a *nested bundle*. A
    Web bundle is never fetched from other web bundles.
 
+## Process web bundle response ## {#process-web-bundle-response}
+
+To <dfn id="concept-process-web-bundle-response">process web bundle
+response</dfn> given [=web bundle=] |web bundle| and [=response=] |response|:
+
 1. If |response|'s [=response/status=] is 200,
 
    1. Set |web bundle|'s [=web bundle/fetched bundle=] to the result of parsing
```

Comment:
Note that `processResponse` is called when response's header list has been received; response body bytes are not fully avaiable yet.

This step passes response's body to the bundle parser (that matches our implementation which supports streaming parsing), but you can only set the parsing result to `fetched bundle` asynchronously.

Reply:
A good point. I'm not sure what is the best way to mention that, but I've added a note to mention that UA might parse the bundle by [incrementally read](https://fetch.spec.whatwg.org/#body-incrementally-read) a body.

Reply:
Could you state that parser runs asynchronously? Something like:

> When the parse algorithm asynchronously completes, set web bundle's fetched bundle to the result...

Reply:
Sure. Clarified that we set items when parse algorithm asynchronously completes. UA can optimize here, but the spec doesn't have to mention the details.